content/docs/0.8.1/volumes-and-nodes/create-volumes.md

```diff
@@ -54,4 +86,30 @@ The above yaml file contains two parts:
         persistentVolumeClaim:
           claimName: longhorn-volv-pvc
     ```
-More examples are available [here.](../../users-guide/examples)
\ No newline at end of file
+More examples are available [here.](../../users-guide/examples)
+
+### Binding Workloads to PVs without a Kubernetes StorageClass
+
+It is possible to use a Longhorn StorageClass to bind a workload to a PV without creating a StorageClass object in Kubernetes.
+
+Since the Storage Class is also a field used to match a PVC with a PV, which doesn't have to be created by a Provisioner, you can create a PV manually with a custom StorageClass name, then create a PVC asking for the same StorageClass name.
+
+When a PVC requests a StorageClass that does not exist as a Kubernetes resource, the PVC will be pending. Kubernetes will bind your PVC to a PV with the same StorageClass name. The StorageClass will be used like a label to find the matching PV, and only existing PVs labeled with the StorageClass name will be used.
```

Comment:
`When a PVC requests a StorageClass that does not exist as a Kubernetes resource, Kubernetes will try to bind your PVC to a PV with the same StorageClass name. `

Reply:
updated